commons/src/components/Tooltip.svelte

```diff
@@ -67,19 +75,23 @@
 
 <button
   class="tooltip-trigger"
-  class:reverse-icon={isTooltipVisible}
   aria-expanded={isTooltipVisible ? "true" : "false"}
-  id={id ? `button-${id.slice(0, 3)}` : ""}
+  id={id ? `tooltip-trigger-${id}` : ""}
   aria-describedby={id}
   aria-label={isTooltipVisible ? "hide email" : "show email"}
   on:click|stopPropagation={(e) => toggleTooltipVisibility(e)}
 >
+  {#if text}
+    <p>{@html text}</p>
```

Comment:
Can you find a way to do this without using `@html` tags? We've been avoiding them if possible because they expose XSS vulnerabilities.

Reply:
Remove the @html modifiers, styling is a bit inconsistent now though.